src/components/NewDraftWizard/WizardForms/WizardJSONSchemaParser.js

```diff
@@ -383,17 +384,14 @@ const FormOneOfField = ({
 
   if (itemValue) {
     switch (lastPathItem) {
-      case "prevStepIndex": {
-        fieldValue = "String value"
-      }
+      case "prevStepIndex":
```

Comment:
I wonder why there is switch instead of if or && when there is only one option for case?

Reply:
This code is not part of this PR (actually from PR https://github.com/CSCfi/metadata-submitter-frontend/pull/323) but I can say that this is a temporary way of writing it in case we have some other fields that face the same error. If so, we can add more `case`. As explained in the comment also:
`// Special handling for Expected Base Call Table > Processing > Complex Processing > Pipeline > Pipe Section > Prev Step Index
// Can be extended to other fields if needed`

We can definitely shorten the code as the way you suggested if later there is no more `case` :)

Reply:
:) ok, thanks.